### DISCUSSION TITLE
[improvement] add per session view

### DISCUSSION THREAD

**Discussion by boly38** [umami-software/umami#1817] [Ideas]
Hi

In real time view, you can click on "visitor" which means database "per session id" related traffic.

Can we imagine a new ui-view for a given session id ?
- It would focus on page view and events for a given session
- the goal is to analyse/understand one enduser-session UX-experience.

And in addition to go deep we could imagine an optional "get session info" url extension to configure per site: 
- on umami sessionid clic: does website have "get session info" url ?
- (if yes) send session Id, and get target website username : enrich ui with real target website username

**Comment by ptman** [comment]
This would indeed be very useful to track how a user behaves in an SPA

Where is it documented what constitues a session? Hash of browser, os, mobile/desktop and browser window resolution?
